### PR TITLE
Standalone apps

### PR DESCRIPTION
**Pull request by adam-abed-abud**
Hi, 

I made a slim version of my standalone applications. They are in "apps/daqdb-top". I'm planning to make the testing automated. 
I also added a README file with some documentation. 

Thanks,
Adam

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -124,7 +124,7 @@ endif()
 
 if(NOT THIN_LIB)
 	set(EXAMPLES ${PROJECT_SOURCE_DIR}/examples)
-	add_subdirectory(${EXAMPLES}/clinode)
+	#add_subdirectory(${EXAMPLES}/clinode)
```
Comment (gjerecze) [inline CMakeLists.txt:127]: Why is this commented out?

File: CMakeLists.txt [hunk 2]
```diff
@@ -173,7 +174,7 @@ if(NOT THIN_LIB)
 		COMMAND ${CMAKE_BUILD_TOOL} functests_thin  # TODO: remove after cmake fix
 		WORKING_DIRECTORY tests/functional_thin
 	)
-	add_dependencies(tests_func_thin clinode daqdb_thin)
+	#add_dependencies(tests_func_thin clinode daqdb_thin)
```
Comment (gjerecze) [inline CMakeLists.txt:177]: Same question here..

File: include/config/Configuration.h [hunk 3]
```diff
@@ -1,27 +0,0 @@
-/**
```
Comment (gjerecze) [inline include/config/Configuration.h:1]: What happened to this file?

File: include/config/Configuration_DAQDB.h [hunk 4]
```diff
@@ -1,27 +0,0 @@
...
-
-#include <daqdb/Options.h>
-#include <sstream>
-
-namespace DaqDB {
-bool readConfiguration(const std::string &configFile, DaqDB::Options &options);
-bool readConfiguration(const std::string &configFile, DaqDB::Options &options,
-                       std::stringstream &ss);
-
-} // namespace DaqDB
```
Comment (gjerecze) [inline include/config/Configuration_DAQDB.h:27]: And to this one?

File: CMakeLists.txt (hunk 1 above)
Comment (adam-abed-abud) [reply CMakeLists.txt]: Restored it, I don't remember why I commented out the "clinode"

File: CMakeLists.txt (hunk 2 above)
Comment (adam-abed-abud) [reply CMakeLists.txt]: Restored it, I don't remember why I commented out the "clinode"

File: include/config/Configuration.h (hunk 3 above)
Comment (adam-abed-abud) [reply include/config/Configuration.h:1]: We renamed it to KVStoreConfig.h

File: include/config/Configuration_DAQDB.h (hunk 4 above)
Comment (adam-abed-abud) [reply include/config/Configuration_DAQDB.h:27]: We renamed it KVStoreConfig.h

**Review by gjerecze — APPROVED** [review]

**Pull request merged**